swift-package/BrainFlowCI/RunCI.sh

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I would add it to CI as commands in yaml, dont see the need for shell script

Reply:
ok

Reply:
Hi. I took a break to work on trying to build the BrainFlow dylibs using Xcode so that they are acceptable to the App Store. I have made progress but I'm not quite there yet. I will return to focusing on the CI cmds in a few days. Later I think I should open a new issue for the App Store build.

Is it easy to build static .a libraries for BrainFlow, instead of .dylibs?